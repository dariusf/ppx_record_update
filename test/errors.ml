
let () =
  [%record
    ()
  ]

let () =
  [%record
    a.b <- 1;
    ()
  ]
