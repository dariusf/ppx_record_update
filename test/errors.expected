let () = [%ocaml.error "expected a field assignment using <- or <~"]
let () = [%ocaml.error "there can be only one root node"]
let () = [%ocaml.error "this value is already assigned"]
let () = (({ a with A.b = 2; b = 1 })[@ocaml.warning "-23"])
